smrf/tests/Lakes/config.ini

```diff
@@ -124,4 +124,4 @@ input_backup:                  True
 [system]
 threading:                     False
 log_level:                     debug
-log_file:                      ./output/log.txt
+#log_file:                      ./output/log.txt
```

Comment:
No more logs in test runs?

Reply:
Just for Lakes, so 2 runs. It's to try and catch any useful output if it does fail in threading. If the issue doesn't crop back up then we can change it back.